setup.cfg

```diff
@@ -1,18 +1,116 @@
-[versioneer]
-VCS = git
-style = pep440
-versionfile_source = napari/_version.py
-versionfile_build = napari/_version.py
-tag_prefix = v
-parentdir_prefix = napari-
+[metadata]
+name = napari
+url = https://napari.org
+download_url = https://github.com/napari/napari
+license = BSD 3-Clause
+license_file = LICENSE
+description = n-dimensional array viewer in Python
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = napari team
+author_email = what_is_it@gmail.com  # FIXME
```

Comment:
Fix me indeed. LOL

Could be the steering council email? I think it's `napari-steering-council@googlegroups.com`. **Edit:** Yep, just verified.

Reply:
btw I don't think it would be too hard to make it `steering-council@napari.org`, which would be pretty badass. =P